### ISSUE TITLE
Problems with Piwik v2.12.1

### ISSUE THREAD

**Issue by pklaus** [kwasib/IntranetSubNetwork#5]
I was getting the following error in the Apache logs with the latest Piwik version 2.12.1:

```
mod_fcgid: stderr: PHP Fatal error:  Call to undefined method Piwik\\IP::N2P() in /var/www/plugins/IntranetSubNetwork/IntranetSubNetwork.php on line 101
```

Had to disable the plugin to get let Piwik do its logging properly again.

Will be looking into it later.

**Comment by kwasib** [comment]
I haven't tested with the latest version of Piwik yet, but I believe this was addressed in f3795b9f735c2cb29301824d783df4a58a24031e.  As of 0.7.1, it should be using Piwik\Network instead of Piwik\IP.

**Comment by pklaus** [comment]
Oops, you're totally right. I just didn't update to 0.7.1. Sorry for the false alarm - works like a charm!
